crates/driver/src/simulator/tenderly/mod.rs

```diff
@@ -0,0 +1,44 @@
+use {crate::logic::eth, thiserror::Error};
+
+mod dto;
+
+#[derive(Debug)]
+pub(super) struct Tenderly {
+    client: reqwest::Client,
+    config: Config,
+}
+
+#[derive(Debug)]
+pub struct Config {
+    pub network_id: eth::NetworkName,
```

Comment:
I think tenderly uses chain IDs and not networks names or versions. Probably run into this in the other PR, so I think its fine to leave this in here.